test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -359,7 +359,7 @@ func findIrqBalanceBannedCPUsVarFromConf(conf string) string {
 		if strings.HasPrefix(line, "#") {
 			continue
 		}
-		if !strings.HasPrefix(line, "IRQBALANCE_BANNED_CPUS") {
+		if !strings.HasPrefix(line, "IRQBALANCE_BANNED_CPULIST") {
```

Comment:
Is this change alone sufficient, because the CPULIST is now contains the list where as the IRQBALANCE_BANNED_CPUS was mask ?,